airbyte-webapp/src/components/JobItem/components/AttemptDetails.tsx

```diff
@@ -61,36 +73,58 @@ const AttemptDetails: React.FC<IProps> = ({
   const minutes = Math.abs(date2.diff(date1, "minute")) - hours * 60;
   const seconds =
     Math.abs(date2.diff(date1, "second")) - minutes * 60 - hours * 3600;
+  const isFailed = attempt.status === Status.FAILED;
 
   return (
     <Details className={className}>
-      <span>{formatBytes(attempt.bytesSynced)} | </span>
-      <span>
-        <FormattedMessage
-          id="sources.countRecords"
-          values={{ count: attempt.recordsSynced }}
-        />{" "}
-        |{" "}
-      </span>
-      <span>
-        {hours ? (
-          <FormattedMessage id="sources.hour" values={{ hour: hours }} />
-        ) : null}
-        {hours || minutes ? (
-          <FormattedMessage id="sources.minute" values={{ minute: minutes }} />
-        ) : null}
-        <FormattedMessage id="sources.second" values={{ second: seconds }} />
-      </span>
-      {configType ? (
+      <div>
+        <span>{formatBytes(attempt.bytesSynced)} | </span>
         <span>
-          {" "}
+          <FormattedMessage
+            id="sources.countEmittedRecords"
+            values={{ count: attempt.totalStats?.recordsEmitted }}
+          />{" "}
           |{" "}
+        </span>
+        <span>
+          <FormattedMessage
+            id="sources.countCommittedRecords"
+            values={{ count: attempt.totalStats?.recordsCommitted }}
```

Comment:
sometimes recordsCommitted will be null. in those cases we should display the world unavailable.